chord_metadata_service/chord/workflows/metadata.py

```diff
@@ -80,6 +84,26 @@ def json_file_input(id_: str, required: bool = True):
     ],
 ))
 
+workflow_set.add_workflow(WORKFLOW_EXPERIMENTS_EXTENDED_JSON, wm.WorkflowDefinition(
+    type="ingestion",
+    name="Bento Experiments JSON-extended",
+    description="This workflow ingests experiments and related files into DRS.",
+    data_type=DATA_TYPE_EXPERIMENT,
+    tags=[DATA_TYPE_EXPERIMENT, "experiment_result"],
+    file="experiments_extended_json.wdl",
+    inputs=[
+        # injected
+        ACCESS_TOKEN_INPUT,
+        DRS_URL_INPUT,
+        KATSU_URL_INPUT,
+        VALIDATE_SSL_INPUT,
+        # user
+        PROJECT_DATASET_INPUT,
+        DIRECTORY_PATH_INPUT,
```

Comment:
I think this should come after `json_document` here, so that the UI has a more logical progression.

Reply:
I tryed and it looks weird to me, it goes project > folder > file. But if you think it is better file > folder, I would change it :)